recipes/foamalgo/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "foamalgo" %}
+{% set version = "0.0.2" %}
+{% set sha256 = "cda69b38c75c867d8037e44b94a238b7d1fd9d7d9bc46760dd32f4128e321a25" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/zhujun98/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - tbb-devel >=2020.1
+    - numpy
+  run:
+    - python
+
+test:
+  imports:
+    - pyfoamalgo
+    - nanmean_image_data
+    - correct_image_data
+    - mask_image_data
+    - AzimuthalIntegrator
+    - geometry
+
+about:
+  home: https://github.com/zhujun98/foamalgo
+  license: GPL 3.0
```

This is not following the spdx format, please change it to
```suggestion
  license: GPL-3.0-or-later
```
or
```suggestion
  license: GPL-3.0-only
```

---

Done! Thanks!